PlagiarismPlugin.php

```diff
@@ -169,88 +199,92 @@ public function callback($hookName, $args) {
+	function getActions($request, $verb)
+	{
+		$router = $request->getRouter();
+		import('lib.pkp.classes.linkAction.request.AjaxModal');
+		return array_merge(
+			$this->getEnabled() ? [
+				new LinkAction(
+					'settings',
+					new AjaxModal(
+						$router->url($request, null, null, 'manage', null, ['verb' => 'settings', 'plugin' => $this->getName(), 'category' => 'generic']),
+						$this->getDisplayName()
+					),
+					__('manager.plugins.settings'),
+					null
+				),
+			] : [],
+			parent::getActions($request, $verb)
+		);
+	}
 
-    /**
-     * @copydoc Plugin::manage()
-     */
-    function manage($args, $request) {
-        switch ($request->getUserVar('verb')) {
-            case 'settings':
-                $context = $request->getContext();
+	/**
+	 * @copydoc Plugin::manage()
+	 */
+	function manage($args, $request)
+	{
+		switch ($request->getUserVar('verb')) {
+			case 'settings':
+				$context = $request->getContext();
 
-                AppLocale::requireComponents(LOCALE_COMPONENT_APP_COMMON, LOCALE_COMPONENT_PKP_MANAGER);
-                $templateMgr = TemplateManager::getManager($request);
-                $templateMgr->registerPlugin('function', 'plugin_url', array($this, 'smartyPluginUrl'));
+				$templateMgr = TemplateManager::getManager($request);
+				$templateMgr->registerPlugin('function', 'plugin_url', [$this, 'smartyPluginUrl']);
 
-                $this->import('PlagiarismSettingsForm');
-                $form = new PlagiarismSettingsForm($this, $context->getId());
+				$form = new PlagiarismSettingsForm($this, $context->getId());
 
-                if ($request->getUserVar('save')) {
-                    $form->readInputData();
-                    if ($form->validate()) {
-                        $form->execute();
-                        return new JSONMessage(true);
-                    }
-                } else {
-                    $form->initData();
-                }
-                return new JSONMessage(true, $form->fetch($request));
-        }
-        return parent::manage($args, $request);
-    }
+				if ($request->getUserVar('save')) {
+					$form->readInputData();
+					if ($form->validate()) {
+						$form->execute();
+						return new JSONMessage(true);
+					}
+				} else {
+					$form->initData();
+				}
+				return new JSONMessage(true, $form->fetch($request));
+		}
+		return parent::manage($args, $request);
+	}
 }
 
 /**
  * Low-budget mock class for \bsobbe\ithenticate\Ithenticate -- Replace the
  * constructor above with this class name to log API usage instead of
  * interacting with the iThenticate service.
  */
-class TestIthenticate {
-	public function __construct($username, $password) {
+class TestIthenticate
+{
+	public function __construct($username, $password)
+	{
 		error_log("Constructing iThenticate: $username $password");
 	}
 
-	public function fetchGroupList() {
+	public function fetchGroupList()
+	{
 		error_log('Fetching iThenticate group list');
 		return array();
```

Comment:
As we are using newer array syntax `[]`, better to keep that same format everywhere like `return []` ;